guides/common/modules/ref_initialization-script-for-provisioning-examples.adoc

```diff
@@ -27,26 +27,6 @@ hammer subscription upload \
 --file ~/$MANIFEST \
 --organization "ACME"
 
-hammer repository-set enable \
```

Comment:
You can't just drop repositories and expect it to be correct. And this is about clients so please revert all changes in this file.

Reply:
Addressed :+1: